.changeset/khaki-hairs-vanish.md

```diff
@@ -0,0 +1,8 @@
+---
+"fluid-framework": patch
+"@fluidframework/map": patch
```

Comment:
Note for @microsoft/fluid-cr-docs follow-up - does not block this PR:

This is awesome and what I would expect from most changesets - the list of packages is small and all of them are part of the "public API."

I've been toying with the idea of using the first package in the front-matter as the primary package for the change, and maybe automatically prefix the header line with the package name (we've done this manually fin past releases), e.g. map: APIs not to be extended have been `@sealed`.

But maybe fluid-framework is always the package that should be highlighted when it's affected - so in this case maybe the title is something like: fluid-framework: Some SharedMap/SharedMap-related APIs have been sealed

Reply:
Another thing - we could include the full list of affected packages in the auto-generated release notes, as a footer. But even with that I think each change needs a "primary package" that it refers to.

Reply:
I guess it depends on who the audience is for the release notes. If it's 1P customers, highlighting `fluid-framework` likely doesn't make sense, and you would want to focus on `map` for this case. But if it's 3P, then that might make sense.

What is the benefit of designating a "primary" package for a change?

Reply:
> I guess it depends on who the audience is for the release notes.

Release notes are for all customers, which admittedly makes it a little tricky. We need to help customers understand whether they care about a change and knowing what package(s) were affected is part of that. Today we manually prefix the title with a package name, but I would love to do something better - just don't know what that is.

> What is the benefit of designating a "primary" package for a change?

Mostly for grouping/sorting, so all Tree-related changes are together for example. Right now everything is sorted manually in the release notes. I don't want to go nuts with metadata - but a few things might help.